source/Calamari.ConsolidateCalamariPackages.Api/IConsolidatedPackageIndex.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Calamari.ConsolidateCalamariPackages.Api
+{
+    public interface IConsolidatedPackageIndex
+    {
+        Package GetPackage(string id);
+
+        public IEnumerable<(string package, string version)> GetAvailablePackages();
+
+        public record Package(string PackageId, string Version, bool IsNupkg, Dictionary<string, FileTransfer[]> PlatformFiles);
+        
+        public record FileTransfer(string Source, string Destination);
```

Comment:
Nit: should these be in the interface?

Reply:
? are they not? Or do you mean as a non-nested class?